lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1134,6 +1116,11 @@ struct FIRModuleContext : public FIRParser {
     return addSymbolEntry(name, SymbolValueEntry(value), loc,
                           insertNameIntoGlobalScope);
   }
+  ParseResult addSymbolEntry(StringRef name, Operation *op, SMLoc loc,
+                             bool insertNameIntoGlobalScope = false) {
+    return addSymbolEntry(name, SymbolValueEntry(op), loc,
```

Comment:
re:being defensive re:portNames, maybe this should insist the operation has that attribute?
(if we had an interface for this, but not presently...)